src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -65,7 +65,7 @@ export const MainFilterList: DbKey[] = [
     "federalState",
 ];
 
-export const subFiltersList: string[] = [
+export const microorganismSubFiltersList: string[] = [
```

Comment:
The underscore keys are only needed in the MatrixDetails so that the keys are unique. 
For example, the microorganism (parent) Salmonella (trigger) has the subfilter Serovar - this combination is unique and occurs only in this way. 
But ALL subfilters of Matrix (parent) are called Matrixdetail. So that they cannot be uniquely assigned to their triggers. That's why the trigger is written behind "martixdetail" to not lose this information.

Reply:
Thx, that makes sense. So "matrixDetail__*" can be read as "matrixDetail for *".